react/Dialog/DialogEffects.spec.ts

```diff
@@ -0,0 +1,211 @@
+import { Theme } from '@material-ui/core'
+
+import { makeOnMount, makeOnUnmount } from './DialogEffects'
+
+const theme = {
+  palette: { background: { paper: '#fff', default: '#000' } }
+} as Theme
+
+const cozybar = document.createElement('div')
+cozybar.style.backgroundColor = '#fff'
+
+const sidebar = document.createElement('div')
+sidebar.style.backgroundColor = '#fff'
+
+const rootModal = document.createElement('div')
+rootModal.style.color = 'rgba(29, 33, 42, 0.9)'
+
+it('should return the correct values with onMount blank', () => {
```

Comment:
nit: a general opinion about the naming of the tests.

Using "return the correct values" does not give any information to the test reader. These tests don't provide intention of the developer. I would recommend less generic term, and caring of the naming specifically to every tests.

ex: here I would rename this test into,
it('should provide a light theme when given mounted theme is blank'

Reply:
I agree